index.html

```diff
@@ -960,8 +599,10 @@
     renderBoundingbox(gl, boundingBoxProgram, modelMatrix, viewMatrix, projectionMatrix,
       entryFramebuffer, exitFramebuffer);
 
-    // Get the step size from the slider and rescale to get into a proper range
-    const stepSize = 1.0 / document.getElementById("stepSize").value;
+    // Get the step size from the slider and rescale to get into a proper range. Note that
+    // the value of the input tag is a string, so we make sure to convert it to a number
+    const nSteps = Number(document.getElementById("numberSteps").value);
+    const stepSize = 1.0 / nSteps;
```

Comment:
Since I renamed the input, there is no need for the extra variable
```suggestion
    const stepSize = 1.0 / Number(document.getElementById("numberSteps").value);
```